src/components/Book/Info.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Info = () => (
+  <div className="info">
+    <p className="categorie">action</p>
+    <p className="title" />
+    <p className="author">Author</p>
+    <div className="creButtons">
+      <p className="comments">Comments</p>
+      <p className="remove">Remove</p>
+      <p className="edit">Edit</p>
+    </div>
+  </div>
+);
```

Comment:
- [ ] This being the book component that is rendered by the booklist which pass props to this component, your book component i.e (info) should take the book data as a prop and renders the title and the author. Pass the props for book data appropriately.